azkaban-common/src/test/resources/azkaban/executor/embedded/innerFlow.job

```diff
@@ -0,0 +1,5 @@
+type=javaprocess
+java.class=azkaban.test.executor.SleepJavaJob
```

Comment:
why don't we reuse flows from https://github.com/azkaban/azkaban/tree/master/azkaban-test/src/test/resources/executions ?